PROVENANCE.md

```diff
@@ -13,16 +13,16 @@ $ pip download --only-binary=:all: launchdarkly-server-sdk
 
 # Download provenance from Github release into same directory
 $ curl --location -O \
-  https://github.com/launchdarkly/python-server-sdk/releases/download/9.2.0/multiple.intoto.jsonl
+  https://github.com/launchdarkly/python-server-sdk/releases/download/9.2.1/multiple.intoto.jsonl
 
 # Run slsa-verifier to verify provenance against package artifacts 
 $ slsa-verifier verify-artifact \
 --provenance-path multiple.intoto.jsonl \
 --source-uri github.com/launchdarkly/python-server-sdk \
-launchdarkly_server_sdk-9.2.0-py3-none-any.whl
+launchdarkly_server_sdk-9.2.1-none-any.whl
```

Comment:
Is there any way to further dictate how release-please updates the version number? We'd want it to update the 9.2.X part here for the purposes of providing an example command for users to run, but it's also removing the `-py3` part which wouldn't be correct